src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/domain/AlertTranslation.kt

```diff
@@ -46,6 +46,15 @@ fun Alert.toAlertModel(auditEvents: Collection<AuditEvent>? = null): AlertModel
   val lastModifiedAuditEvent = events.firstOrNull { it.action == AuditEventAction.UPDATED }
   val lastActiveToSetAuditEvent = events.takeIf { activeTo != null }
     ?.firstOrNull { it.action == AuditEventAction.UPDATED && it.activeToUpdated == true }
+  val deactivationEvent = events.firstOrNull {
+    it.action == AuditEventAction.INACTIVE ||
+      (it.activeToUpdated == true && it.actionedAt.toLocalDate() == activeTo)
```

Comment:
This bit is essentially covered by the lastActiveToSetAuditEvent above. Could change deactivationEvent to inactiveEvent and do the full cascade below as inactiveEvent ?: lastActiveToSetAuditEvent ?: lastModifiedAuditEvent ?: createdAuditEvent to show the precedence in full

Minor and certainly not required so happy to approve as is